driver/bpf/plumbing_helpers.h

```diff
@@ -125,7 +140,7 @@ static __always_inline long bpf_syscall_get_nr(void *ctx)
 static __always_inline unsigned long bpf_syscall_get_argument_from_args(unsigned long *args,
 									int idx)
 {
-	unsigned long arg;
+	unsigned long arg = 0;
```

Comment:
[ultra-nit] With this assignment we could remove the `else` block in this function.